core/vm/operations_verkle.go

```diff
@@ -23,15 +23,16 @@ import (
 )
```

Comment:
Maybe this is due to the PR being WIP, but we should also mark as `isFill` any case where a new account is created.

Reply:
yeah this version is only about SSTORE for now, in order to validate a few things while keeping stuff simple.